cockatrice/src/messagelogwidget.cpp

```diff
@@ -251,7 +251,7 @@ void MessageLogWidget::logCreateArrow(Player *player,
 
 void MessageLogWidget::logCreateToken(Player *player, QString cardName, QString pt)
 {
-    appendHtmlServerMessage(tr("%1 creates token: %2%3.")
+    appendHtmlServerMessage(tr("%1 creates token: %2 %3.")
```

Comment:
%3 can be empty, this is why the space isn't here by default

Reply:
Checking the arguments again, `%2%3` seems totally fine here and it's actually not connected to this issue.

Will revert. Thank you!